qiita_db/handlers/tests/test_artifact.py

```diff
@@ -66,9 +66,13 @@ def test_get_artifact(self):
         path_builder = partial(join, db_test_raw_dir)
         exp_fps = {
             "raw_forward_seqs":
-                [path_builder('1_s_G1_L001_sequences.fastq.gz')],
+                [{'filepath': path_builder('1_s_G1_L001_sequences.fastq.gz'),
+                  'size': 58}],
             "raw_barcodes":
-                [path_builder('1_s_G1_L001_sequences_barcodes.fastq.gz')]}
+                [{'filepath': path_builder(
+                  '1_s_G1_L001_sequences_barcodes.fastq.gz'),
+                  'size': 58}]
```

Comment:
I'm wondering if it would be good to describe the nature of these two 58s and the 1093210 below with a comment. Intentionally small file size and intentionally larger file size and all that.

Reply:
I don't think there is anything behind those numbers, they are just what's preloaded in the database.

Reply:
Ok.